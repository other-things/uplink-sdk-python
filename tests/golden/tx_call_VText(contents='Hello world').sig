r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 5332284521949599856840544988092336962754555761566548837029979265767462135386
