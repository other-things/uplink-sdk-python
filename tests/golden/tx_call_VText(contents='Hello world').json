{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "contents": "Hello world",
                        "tag": "VText"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AEw1MzMyMjg0NTIxOTQ5NTk5ODU2ODQwNTQ0OTg4MDkyMzM2OTYyNzU0NTU1NzYxNTY2NTQ4ODM3MDI5OTc5MjY1NzY3NDYyMTM1Mzg2"
}